newton/_src/solvers/sph/sampled_boundary_kernels.py

```
@@ -0,0 +1,358 @@
+
+    force = wp.vec3(0.0)
+    width = periodic_domain_max - periodic_domain_min
+
+    for ox in range(-1, 2):
+        for oy in range(-1, 2):
+            for oz in range(-1, 2):
+                if use_periodic_domain or (ox == 0 and oy == 0 and oz == 0):
+                    shift = wp.vec3(float(ox) * width[0], float(oy) * width[1], float(oz) * width[2])
+                    x_query = x_i + shift
+                    query = wp.hash_grid_query(grid, x_query, h_i)
+                    j = int(0)
```

Ruff (0.15.18)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)